src/org/zaproxy/zap/spider/SpiderTask.java

```diff
@@ -311,9 +311,9 @@ private ExtensionHistory getExtensionHistory() {
 	 * @return the response http message
 	 * @throws HttpException the http exception
```

Comment:
That exception is already included by the declaration of IOException, same for other exceptions re-thrown.